kafka/coordinator/consumer.py

```diff
@@ -751,6 +756,13 @@ def _send_offset_fetch_request(self, partitions):
 
     def _handle_offset_fetch_response(self, future, response):
         offsets = {}
+        if response.API_VERSION > 1:
+            error_type = Errors.for_code(response.error_code)
+            if error_type is not Errors.NoError:
+                log.debug("Group %s failed to fetch offsets: %s", self.group_id,
+                          error_type)
+                future.failure(error_type())
+                return
```

Comment:
@royfranco
NIT:
1. You can add this block before the initialization of the local `offsets` dict.
2. The code below first instantiate the error_type (by using it as a callable : `()`) and then log it and pass it to the future.